app/Services/PasswordService.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Services;
+
+use Illuminate\Support\Facades\Hash;
+use App\User;
+
+class PasswordService 
+{
+	public function edit($id)
+	{
+		$user = User::findOrFail($id);
+
+		return $user;
+	}
+
+	public function changePassword($data, $id)
+	{
+		try {
+			$user = User::findOrFail($id);
+			$current_password = $user->password;
```

Comment:
```suggestion
			$currentPassword = $user->password;
```
camelCase nè, e chưa sài thằng phpcs à =))